librna/ViennaRNA/static/misc/dna_mathews1999.hex

```diff
@@ -52,10 +52,10 @@
   0x2d, 0x32, 0x30, 0x20, 0x20, 0x20, 0x2d, 0x36, 0x30, 0x20, 0x20, 0x2d,
   0x31, 0x33, 0x30, 0x20, 0x20, 0x2d, 0x31, 0x36, 0x30, 0x20, 0x20, 0x20,
   0x20, 0x20, 0x30, 0x0a, 0x20, 0x20, 0x2d, 0x31, 0x39, 0x30, 0x20, 0x20,
-  0x2d, 0x32, 0x32, 0x30, 0x20, 0x20, 0x20, 0x44, 0x45, 0x46, 0x20, 0x20,
+  0x2d, 0x32, 0x32, 0x30, 0x20, 0x20, 0x20, 0x2d, 0x35, 0x30, 0x20, 0x20,
```

Comment:
So this here for example is a change the Vienna team made?

Can anyone explain why they did that? I get that the energy models need to be adjusted - but wouldn't you rename the model to something like "mathews1999.2"? 

Did they mention the reason why they changed it?

Reply:
Yes, this is a change that the ViennaRNA team made. These `.hex`-files are generated from the `.par`-files in `gapc/librna/paramfiles/`, which were adjusted in the v2.4.18 release. The ViennaRNA commit message for those changes was: "replaced DEF placeholders in parameter files with their actual value -50" (see Changelog). So the changes in the `.hex`-files are probably coming from those new values. What this means, why they changed the DEF placeholders to -50 and why they didn't rename the model I cannot answer.

Reply:
for a reason of change see https://github.com/ViennaRNA/ViennaRNA/issues/115

Reply:
Ah, this makes sense, thank you!